src/extension.ts

```diff
@@ -51,6 +52,7 @@ let taskProvider: vscode.Disposable;
 const log = logging.createLogger('extension');
 
 const multiRootModeKey = 'cmake:multiRoot';
+const multiProjectModeKey = 'cmake:multiProject';
```

Comment:
I don't see this used in `package.json`. Do we need it?

Reply:
If we consider multi-project and multi-root to be the same thing when we run configure all, then we don't need the other key. 
I only kept the telemetry separate for multi-root and multi-project so we know about the customers who are actually using the feature.

Reply:
This isn't for telemetry.  You can still send telemetry for multiProject without this.  The "setContext" keys are for enabling/disabling features declared in `package.json`.  Setting `cmake:multiProject` does not enable or disable any features as far as I can tell.